xarray/core/ops.py

```diff
@@ -465,3 +465,7 @@ def inject_all_ops_and_reduce_methods(cls, priority=50, array_only=True):
             setattr(cls, name, _values_method_wrapper(name))
 
     inject_reduce_methods(cls)
+
+def tensordot(a, b, axes):
```

Comment:
I think you can simply write `tensordot = _dask_or_eager_func('tensordot', n_array_args=2)`